nat-lab/tests/config.py

```diff
@@ -21,6 +21,8 @@ def get_root_path(path: str) -> str:
 WINDOWS_1_VM_IP = "10.55.0.13"
 WINDOWS_2_VM_IP = "10.55.0.14"
 
+VM_TCP_CONN_TIMEOUT = 10
```

Comment:
It would be nice to know, either from naming or a comment, what unit this is.  I'm assuming it's in seconds?

Reply:
correct, will add!